core/migrations/migration.py

```diff
@@ -0,0 +1,32 @@
+from typing import Callable
+
+CODE_DB_VERSION = 2
+
+
+class MigrationManager:
+    MIGRATIONS: list[Callable] = []
+
+    def __init__(self):
+        self.connect_to_db()
+
+    def connect_to_db(self):
+        raise NotImplementedError
+
+    def update_db_version(self, version: int):
+        raise NotImplementedError
+
+    def migrate_to_latest(self, stop_at: int = None):
```

Comment:
Why don't rely on migration name which already includes a number?

Reply:
The functions are arbitrarily named, so we wanna calculate them from the order they've been pushed in